### ISSUE TITLE
typescript/member-ordering issue : 

### ISSUE THREAD

**Issue by bichikim** [nzakas/eslint-plugin-typescript#97]
### the filed is function type
* in class
````javascript
  private _required:boolean
  private _typeChecker:(data:any)=>boolean
  constructor(validator:(data:any)=>boolean){ 
  ^^^^^^^^^^^ error
    this._required = false
    this._typeChecker = validator
  }
  check(data:any):boolean{
    if(_.isNil(data)){
      return !this._required
    }
    return this._typeChecker(data)
  }
````
````javascript
  private _required:boolean
  constructor(validator:(data:any)=>boolean){ 
    this._required = false
    this._typeChecker = validator
  }
 private _typeChecker:(data:any)=>boolean
 ^^^^^^^ error
  check(data:any):boolean{
    if(_.isNil(data)){
      return !this._required
    }
    return this._typeChecker(data)
  }
````
 * in eslintrc.js
````javascript
    'typescript/member-ordering': ['error', {
      'classes': [
        'field', 'constructor', 'method'
      ],
    }],
````
The field should be a filed not method

**Comment by macklinu** [comment]
According to the rule's current implementation, changing the code to look like this will resolve the linter error:

```ts
class Foo {
    private required: boolean;
    constructor(validator: (data: any) => boolean) {
        this.typeChecker = validator;
    }
    private typeChecker: (data: any) => boolean;
    check(data: any): boolean {
        return this.typeChecker(data);
    }
}
```

Based on the rule configuration, I would expect to have the private fields located next to each other, which I believe is the original intention of this issue.

Could a contributor/collaborator please chime in and confirm if this is a bug or expected behavior? If it's a bug, I will push a change to group fields together (regardless if they are function types) to #116 - glad to discuss here or there.
